doc/source/changes/version_0_25.rst.inc

```diff
@@ -5,6 +5,67 @@
   It contains a link to open the documentation, a shortcut to launch the user interface in edition mode
   and a shortcut to update larrayenv. Closes issue:`281`.
 
+* implemented array.align(other_array) which makes two arrays compatible with each other (by making all common axes
+  compatible). This is done by adding, removing or reordering labels for each common axis according to the join method:
```

Comment:
... compatible). The fill value for missing labels defaults to nan. This is done by ...